Image_Viewer/theme.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+class Theme:
+    def __init__(self):
+        themeDictionary = self.read_theme()
+        self.themes = themeDictionary["themes"]
+        self.current_theme=themeDictionary["lastTheme"]
+        
+    
+        
+    
+    def set_theme(self, background, foreground, buttonBackground):
+            self.root.configure(bg=background)
+            self.name_label.configure(bg=background, fg=foreground)
+            self.display.configure(bg=background)
+            self.move_left_button.configure(bg=buttonBackground, fg=foreground)
+            self.move_right_button.configure(bg=buttonBackground, fg=foreground)
+            
+
+    def read_theme(self):
+        with open('Image_Viewer/themes.json', 'r') as openfile:
+            json_object = json.load(openfile)
+            return json_object
+    
+    def write_theme(self):
+        themeDictionary = {"themes":self.themes,"lastTheme":self.current_theme}
```

Comment:
I'd suggest putting these onto separate lines just to help make it a bit easier to read:
```suggestion
        themeDictionary = {
            "themes": self.themes,
            "lastTheme": self.current_theme
         }
```